hostfactory/host_provider/generateWeightedTemplates.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+export HF_LOGDIR=/tmp/log
+export HF_CONFDIR=$HF_TOP/conf
+export HF_WORKDIR=/tmp/work
+mkdir -p $HF_LOGDIR
+mkdir -p $HF_WORKDIR
+cat <<EOF >/tmp/input.json
+{}
+EOF
+export PRO_LOG_DIR=${HF_LOGDIR}
+export PRO_CONF_DIR=${HF_CONFDIR}/providers/azurecc
+export PRO_DATA_DIR=${HF_WORKDIR}
+
+env > /tmp/invoke2.env
+
+venv_path=$HF_TOP/$HF_VERSION/providerplugins/azurecc/venv/bin
+scriptDir=`dirname $0`
+export PYTHONPATH=$PYTHONPATH:$scriptDir/src
+. $venv_path/activate
+$venv_path/python3 -m cyclecloud_provider generate_templates -f /tmp/input.json 2>> /tmp/template_generate.out
+rm -rf $HF_LOGDIR
+rm -rf $HF_WORKDIR
+exit 0
```

Comment:
This is swallowing errors. You should save the exit from line 20 and then exit with that